tests/lookup/tests.py

```diff
@@ -668,6 +673,15 @@ def test_values_list(self):
         with self.assertRaises(TypeError):
             Article.objects.values_list("id", "headline", flat=True)
 
+    def test_values_list_flat_empty_warning(self):
```

Comment:
We should make sure the error is being tested after the deprecation, perhaps:
```suggestion
    # RemovedInDjango70Warning: When the deprecation ends, replace with:
    # def test_values_list_flat_empty_error(self):
    #     msg = "'flat' is not valid when values_list is called with no fields."
    #     with self.assertRaisesMessage(TypeError, msg):
    #         Article.objects.values_list(flat=True)
    def test_values_list_flat_empty_warning(self):
```